### PR TITLE
[fix #104]clean br command args

### LINKED ISSUE #104: Clean TiKV-BR command args
### 1. Describe the bug
1. There are many args in tikv-br and some of them are useless or not supported well. Please remove/hide them.
2. Some help description is not suitable, for example, `tikv-br help`  display `br is a tool for TiDB/TiKV cluster...`, please check all hekp descriptions.

### 2. Minimal reproduce step (Required)
tikv-br help
tikv-br backup help
tikv-br restore help
tikv-br backup raw --help
tikv-br restore raw --help

### 3. What did you see instead (Required)
1. There are many args in tikv-br and some of them are useless or not supported well. Please remove/hide them.
2. Some help description is not suitable, for example, `tikv-br help`  display `br is a tool for TiDB/TiKV cluster...`, please check all hekp descriptions.

### 4. What did you expect to see? (Required)
all command args should be supported well, else remove them.
### 5. What is your migration tool and TiKV version? (Required)
- TiKV BR:

### PR DESCRIPTION
**Pull request by haojinming**
Signed-off-by: haojinming <jinming.hao@pingcap.com>

### What problem does this PR solve?

Issue Number: close #104 
Problem Description: 
 clean command args as descriped in https://pingcap.feishu.cn/docs/doccnCay8PSAR5uiw6RhlXEVz7b.
### Code changes

- Has exported function/method change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Manual test (add detailed scripts or steps below)
-
### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: br/pkg/task/backup_raw.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ func DefineRawBackupFlags(command *cobra.Command) {
 
 	// This flag can impact the online cluster, so hide it in case of abuse.
 	_ = command.Flags().MarkHidden(flagRemoveSchedulers)
+
+	_ = command.Flags().MarkHidden(flagRemoveSchedulers)
```
Comment (pingyu) [inline br/pkg/task/backup_raw.go:54]: Duplicated with above line.

File: br/pkg/task/backup.go [hunk 2]
```diff
@@ -47,6 +47,10 @@ func DefineBackupFlags(flags *pflag.FlagSet) {
 		"disable the balance, shuffle and region-merge schedulers in PD to speed up backup")
 	// This flag can impact the online cluster, so hide it in case of abuse.
 	_ = flags.MarkHidden(flagRemoveSchedulers)
+	_ = flags.MarkHidden(flagBackupTimeago)
```
Comment (pingyu) [inline br/pkg/task/backup.go:50]: Consider also hide `flagCompressionType`.

File: br/pkg/task/backup_raw.go (hunk 1 above)
Comment (pingyu) [reply br/pkg/task/backup_raw.go:54]: Consider also hide `flagCompressionType`. Just one default compress type is enough.

File: br/pkg/task/common.go [hunk 3]
```diff
@@ -82,19 +82,25 @@ const (
...
 	flags.String(flagKey, "", "Private key path for TLS connection")
 	flags.Uint(flagChecksumConcurrency, variable.DefChecksumTableConcurrency, "The concurrency of table checksumming")
+	_ = flags.MarkHidden(flagSendCreds)
+	_ = flags.MarkHidden(flagCA)
+	_ = flags.MarkHidden(flagCert)
+	_ = flags.MarkHidden(flagKey)
 	_ = flags.MarkHidden(flagChecksumConcurrency)
 
 	flags.Uint64(flagRateLimit, unlimited, "The rate limit of the task, MB/s per node")
+	_ = flags.MarkHidden(flagRateLimit)
```
Comment (pingyu) [inline br/pkg/task/common.go:98]: Rate limit is necessary to limit impact to normal work load.

File: br/pkg/task/common.go [hunk 4]
```diff
@@ -82,19 +82,25 @@ const (
...
 	flags.Bool(flagChecksum, true, "Run checksum at end of task")
 	flags.Bool(flagRemoveTiFlash, true,
 		"Remove TiFlash replicas before backup or restore, for unsupported versions of TiFlash")
-
+	_ = flags.MarkDeprecated(flagRemoveTiFlash,
```
Comment (pingyu) [inline br/pkg/task/common.go:102]: Seems we can remove this flag. We have nothing to do with TiFlash.

File: br/pkg/task/backup_raw.go (hunk 1 above)
Comment (haojinming) [reply br/pkg/task/backup_raw.go:54]: ok, make sense.

File: br/pkg/task/common.go (hunk 3 above)
Comment (haojinming) [reply br/pkg/task/common.go:98]: In ccb scenario, we need do back/restore as fast as possible, so we may not need limit the speed as the discussion with QA team and Xiaoguang. After we test it enough, then we enable this.

File: br/pkg/task/common.go (hunk 4 above)
Comment (haojinming) [reply br/pkg/task/common.go:102]: Ok, removed.

File: br/pkg/task/backup.go (hunk 2 above)
Comment (haojinming) [reply br/pkg/task/backup.go:51]: done

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**